actionpack/test/controller/mime/respond_to_test.rb

```diff
@@ -781,4 +781,11 @@ def test_variant_negotiation_without_block
     assert_equal "text/html", @response.content_type
     assert_equal "phone", @response.body
   end
+
+  def test_variant_from_variant_parameter
+    get :format_any_variant_any, params: { format: "js", variant: "tablet" }
```

Comment:
Take a look at the other tests in the file, `format` isn't part of the `params`(and variant shouldn't be either).